Error building Keyman Engine on Ubuntu

**Describe the bug**
I am unable to compile the KMEA/KMAPro projects. As far as I can tell, I meet the prerequisites on Ubuntu: 
- using npm : 6.14.4
- using node : v10.19.0

**To Reproduce**
Steps to reproduce the behavior:
1. `cd keyman/android` 
2. `./build.sh -debug`
3. Observe error at `lerna bootstrap` target

**Expected behavior**
I expected the build.sh script to complete with no errors.

**Complete Error Log**
```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'run', 'bootstrap' ]
2 info using npm@6.14.4
3 info using node@v10.19.0
4 verbose run-script [ 'prebootstrap', 'bootstrap', 'postbootstrap' ]
5 info lifecycle root@~prebootstrap: root@
6 info lifecycle root@~bootstrap: root@
7 verbose lifecycle root@~bootstrap: unsafe-perm in lifecycle true
8 verbose lifecycle root@~bootstrap: PATH: /usr/share/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/wlane/projects/keyman/node_modules/.bin:../node_modules/.bin:/usr/local/cuda-10.1/bin:/usr/local/cuda-10.1/bin:/usr/local/cuda-10.1/bin:/home/wlane/anaconda3/bin:/home/wlane/anaconda3/condabin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/home/wlane/Android/Sdk/tools:/home/wlane/Android/Sdk/tools:/usr/lib/jvm/java-11-openjdk-amd64/bin
9 verbose lifecycle root@~bootstrap: CWD: /home/wlane/projects/keyman
10 silly lifecycle root@~bootstrap: Args: [ '-c', 'lerna bootstrap -- --no-optional' ]
11 silly lifecycle root@~bootstrap: Returned: code: 2  signal: null
12 info lifecycle root@~bootstrap: Failed to exec bootstrap script
13 verbose stack Error: root@ bootstrap: `lerna bootstrap -- --no-optional`
13 verbose stack Exit status 2
13 verbose stack     at EventEmitter.<anonymous> (/usr/share/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:198:13)
13 verbose stack     at ChildProcess.<anonymous> (/usr/share/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:198:13)
13 verbose stack     at maybeClose (internal/child_process.js:982:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
14 verbose pkgid root@
15 verbose cwd /home/wlane/projects/keyman
16 verbose Linux 5.4.0-33-generic
17 verbose argv "/usr/bin/node" "/usr/bin/npm" "run" "bootstrap"
18 verbose node v10.19.0
19 verbose npm  v6.14.4
20 error code ELIFECYCLE
21 error errno 2
22 error root@ bootstrap: `lerna bootstrap -- --no-optional`
22 error Exit status 2
23 error Failed at the root@ bootstrap script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 2, true ]
~                                                                                                                               
~                                                                                                                               
~                                                                                                                               
"~/.npm/_logs/2020-06-19T02_18_28_381Z-debug.log" 34L, 2292C   
```
--

Thanks for taking a look! here is the console output I'm seeing:
```
(base) wlane@wlane-MS-7B90:~/projects/keyman/android$ ./build.sh --debug
Build KMEA and KMAPro:
Build KMEA

DO_BUILD: true
DO_COPY: true
DO_TEST: true
NO_DAEMON: false
DEBUG_BUILD: 
EMBED_BUILD: -embed
KMW_PATH: 

Building keyman web engine
audited 757 packages in 3.681s

24 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities


> root@ bootstrap /home/wlane/projects/keyman
> lerna bootstrap -- --no-optional

lerna notice cli v3.21.0
lerna info Bootstrapping 13 packages
lerna info Installing external dependencies
lerna info hoist Pruning hoisted dependencies
lerna info hoist Finished pruning hoisted dependencies
lerna info hoist Finished bootstrapping root
lerna info Symlinking packages and binaries
lerna info lifecycle @keymanapp/models-wordbreakers@14.0.95~prepublish: @keymanapp/models-wordbreakers@14.0.95

> @keymanapp/models-wordbreakers@14.0.95 prepublish /home/wlane/projects/keyman/common/models/wordbreakers
> npm run build


> @keymanapp/models-wordbreakers@14.0.95 build /home/wlane/projects/keyman/common/models/wordbreakers
> tsc

lerna info lifecycle @keymanapp/web-environment@14.0.95~prepare: @keymanapp/web-environment@14.0.95

> @keymanapp/web-environment@14.0.95 prepare /home/wlane/projects/keyman/resources/web-environment
> gosh ./build.sh

./build.sh: 5: Bad substitution
./build.sh: 31: ./../build/build-utils.sh: Syntax error: "(" unexpected
lerna info lifecycle @keymanapp/web-environment@14.0.95~prepare: Failed to exec prepare script
lerna ERR! lifecycle "prepare" errored in "@keymanapp/web-environment", exiting 2
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! root@ bootstrap: `lerna bootstrap -- --no-optional`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the root@ bootstrap script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/wlane/.npm/_logs/2020-06-19T02_18_28_381Z-debug.log
Build environment setup error detected!  Please ensure Node.js is installed!

ERROR: keymanweb build failed. Exiting


ERROR: KMEA/build.sh failed
```

--- a/resources/web-environment/build.sh
+++ b/resources/web-environment/build.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 ## START STANDARD BUILD SCRIPT INCLUDE
 # adjust relative paths as necessary

